### PR TITLE
Added examples

### PR DESCRIPTION
**Pull request by stavrospapadopoulos**

### CODE REVIEW

**Comment by jakebolewski** [comment]
LGTM, needs to be rebased

File: examples/reading_sparse_layouts.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    # Create the (empty) array on disk.
+    tiledb.SparseArray.create(array_name, schema)
+
+
+def write_array():
+    ctx = tiledb.Ctx()
+    # Open the array and write to it.
+    with tiledb.SparseArray(ctx, array_name, mode='w') as A:
+        # Write some simple data to cells (1, 1), (2, 4) and (2, 3).
+        I, J = [1, 1, 2, 1, 2, 2], [1, 2, 2, 4, 3, 4]
```
Comment (tdenniston) [inline examples/reading_sparse_layouts.py]: This writes 6 cells, the comment is wrong

File: examples/reading_dense_layouts.py [hunk 2]
```diff
@@ -68,31 +63,58 @@ def write_array():
...
 
     # Open the array and read from it.
     with tiledb.DenseArray(ctx, array_name, mode='r') as A:
         # Get non-empty domain
-        print(A.nonempty_domain())
+        print("Non-empty domain: {}".format(A.nonempty_domain()))
 
         # Slice only rows 1, 2 and cols 2, 3, 4.
-        data = A.query(attrs=["a"], coords=True)[1:3, 2:5]
+        data = A.query(attrs=["a"], order=order, coords=True)[1:3, 2:5]
```
Comment (tdenniston) [inline examples/reading_dense_layouts.py]: Worth adding a comment that the `.query()` syntax is required in order to specify coords for dense arrays and the ordering.

File: examples/writing_dense_padding.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                                attrs=[tiledb.Attr(ctx, name="a", dtype=np.int32)])
+
+    # Create the (empty) array on disk.
+    tiledb.DenseArray.create(array_name, schema)
+
+def write_array():
+    ctx = tiledb.Ctx()
+    # Open the array and write to it.
+    with tiledb.DenseArray(ctx, array_name, mode='w') as A:
+        # Write to [2,3], [1,2]
```
Comment (tdenniston) [inline examples/writing_dense_padding.py]: comment is wrong

**Pull request merged**